src/style.css

```diff
@@ -0,0 +1,128 @@
+.page-html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+
+  color: #060b35;
+}
+
+.reset {
+  margin: 0;
+}
```

Comment:
[Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) the 11th point